### PR TITLE
Feature/home screen UI

### PR DESCRIPTION
**Pull request by zhenyabigel**
added ui of home screen, account and transaction cards, screen orientation

### CODE REVIEW

File: app/src/main/java/by/zhenyabigel/bankingapplication/model/Transaction.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package by.zhenyabigel.bankingapplication.model
+
+import androidx.annotation.DrawableRes
```
Comment (Razor0744) [inline app/src/main/java/by/zhenyabigel/bankingapplication/model/Transaction.kt]: Unused imports

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+import by.zhenyabigel.bankingapplication.ui.theme.Green
+import by.zhenyabigel.bankingapplication.ui.theme.LightGray
+import by.zhenyabigel.bankingapplication.ui.theme.Maroon
+import by.zhenyabigel.bankingapplication.ui.theme.Yellow
+import by.zhenyabigel.bankingapplication.ui.theme.robotoFontFamily
+
+@Composable
+fun HomeScreen() {
+    val accounts = cards
+    val transactions = transactions
```
Comment (Razor0744) [inline app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: After a variable need a space line

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+        AccountSection(accounts[1])
+        Spacer(modifier = Modifier.height(16.dp))
+        TransactionSection(transactions)
+        Spacer(modifier = Modifier.height(16.dp))
+        Footer()
+    }
+}
+
+
+@Composable
```
Comment (Razor0744) [inline app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: should be in another file

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+                Image(
+                    painter = painterResource(id = R.drawable.ic_baseline_arrow),
+                    contentDescription = "image arrow forward"
+                )
+            }
+        }
+    }
+}
+
+@Composable
```
Comment (Razor0744) [inline app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: same

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+                            thickness = 0.7.dp,
+                            modifier = Modifier
+                                .padding(horizontal = 16.dp))
+                }
+            }
+        }
+    }
+}
+
+fun getColorForStatus(status: String): Color {
```
Comment (Razor0744) [inline app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: and this

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+    return when (status) {
+        "Executed" -> Green
+        "In progress" -> Yellow
+        "Declined" -> Maroon
+        else -> Color.Gray // Цвет по умолчанию
+    }
+}
+@Composable
```
Comment (Razor0744) [inline app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: this also in another file

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+                    painter = painterResource(id = R.drawable.ic_baseline_arrow_dark),
+                    contentDescription = "image arrow forward"
+                )
+            }
+
+        }
+    }
+}
+
+@Composable
```
Comment (Razor0744) [inline app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: same

File: app/build.gradle.kts
Comment (zhenyabigel) [inline app/build.gradle.kts:1]: wow

File: app/src/main/java/by/zhenyabigel/bankingapplication/MainActivity.kt
Comment (zhenyabigel) [inline app/src/main/java/by/zhenyabigel/bankingapplication/MainActivity.kt:1]: lol

File: app/src/main/java/by/zhenyabigel/bankingapplication/model/Transaction.kt (hunk 1 above)
Comment (zhenyabigel) [reply app/src/main/java/by/zhenyabigel/bankingapplication/model/Transaction.kt]: I did
> Unused imports

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt (hunk 2 above)
Comment (zhenyabigel) [reply app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: I did

> After a variable need a space line

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt (hunk 3 above)
Comment (zhenyabigel) [reply app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: I did

> should be in another file

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt (hunk 4 above)
Comment (zhenyabigel) [reply app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: I did

> same

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt (hunk 5 above)
Comment (zhenyabigel) [reply app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: I did

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt (hunk 6 above)
Comment (zhenyabigel) [reply app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: I did

File: app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt (hunk 7 above)
Comment (zhenyabigel) [reply app/src/main/java/by/zhenyabigel/bankingapplication/screen/HomeScreen.kt]: I did

**Pull request merged**